ckanext/video/theme/assets/video-view.css

```diff
@@ -0,0 +1,16 @@
+.video-view-container {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    padding-top: 56.25%; /* 16:9 Aspect Ratio (divide 9 by 16 = 0.5625) */
```

Comment:
This is the only thing I'm not 100% on, why padding and how this interacts with other elements. 

This might be interesting: https://jakearchibald.com/2022/img-aspect-ratio/

```
.aspect-ratio-demo {
  aspect-ratio: 16 / 9;
}
```